linux_os/guide/services/deprecated/package_telnetd_removed/rule.yml

```diff
@@ -2,18 +2,24 @@ documentation_complete: true
 
 title: 'Uninstall the telnet server'
 
-description: 'The telnet daemon should be uninstalled.'
+description: |-
+        The telnet daemon should be uninstalled.
 
-rationale: 'telnet allows clear text communications, and does not protect any data transmission between client and server. Any confidential data can be listened and no integrity checking is made.'
+rationale: |-
+        <tt>telnet</tt> allows clear text communications, and does not protect
+        any data transmission between client and server. Any confidential data
+        can be listened and no integrity checking is made.'
 
 severity: high
 
 identifiers:
+    cce@rhel6: 82447-4
     cce@rhel7: 82352-6
+    cce@rhel8: 82448-2
```

Comment:
@shawndwells please return these CCE's to the pool. There is no telnetd package in RHEL. It's `telnet-server`.